#goal definition
sensor_msgs/PointCloud2 tgt_cloud
sensor_msgs/PointCloud2 cloud_plane
float32[4] cloud_plane_coef
---
#result definition
bool success
---
#feedback
bool object_moved
